web/src/components/storage/ConfigureDeviceMenu.tsx

```diff
@@ -79,41 +68,32 @@ const DisksDrillDownMenuItem = ({
     : _("Select a disk to define partitions");
 
   return (
-    <MenuItem
-      itemId="group:disks-menu"
-      direction="down"
+    <MenuButtonItem
+      aria-label={_("Add device menu")}
       isDisabled={isDisabled}
       description={isDisabled ? disabledDescription : enabledDescription}
-      drilldownMenu={
-        <DrilldownMenu id="disks-menu">
-          <MenuItem itemId="group:disks-menu-back" direction="up">
-            {_("Back")}
-          </MenuItem>
-          <Divider />
-          {devices.map((device) => (
-            <MenuItem
-              key={device.sid}
-              itemId={device.sid}
-              description={<MenuDeviceDescription device={device} />}
-              onClick={() => onDeviceClick(device.name)}
-            >
-              <Split hasGutter>
-                {deviceLabel(device)}
-                <Flex columnGap={{ default: "columnGapXs" }}>
-                  {device.systems.map((s, i) => (
-                    <Label key={i} isCompact>
-                      {s}
-                    </Label>
-                  ))}
-                </Flex>
-              </Split>
-            </MenuItem>
-          ))}
-        </DrilldownMenu>
-      }
+      upProps={{ label: title }}
```

Comment:
I like having the original label as some kind of "header" for the drilled menu. It allows to retain the context. But...

I wonder whether we should also display the description of whether that would be too much.

I'm not sure whether we should still show the word "back" somewhere. Clicking on "select disk" to actually go back from that action may not be optimal.

Reply:
Restored the old "back" word. It was agreed on a video call.